.rubocop.yml

```diff
@@ -13,7 +13,7 @@ Metrics/BlockNesting:
 # Offense count: 5
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 1916
+  Max: 2000
```

Comment:
Apart of others I've a plan to create a class for "lan item" and then use a kind of container with meaningful iterators instead of "LanItems" that should help with all those "GetItem.." and "SetItem..." and "GetCurrent..." and "SetCurrent...". However, I'd like to do these changes for SLE13.